serverless/README.md

```diff
@@ -32,17 +34,41 @@ should contain the log state files, e.g.:
 $ export LOG_DIR="/tmp/mylog"
 ```
 
+`sequence` and `client` require the log public key to be provided.
+
+This is supplied by providing the path to the key file using `--public_key` 
+or by setting the `SERVERLESS_LOG_PUBLIC_KEY` environment variable
+
+`integrate` requires the log public and private keys to be provided.
+
+These are supplied by providing the path to the key files using 
+`--public_key` and `--private_key` or by setting the
+ `SERVERLESS_LOG_PUBLIC_KEY` and `SERVERLESS_LOG_PRIVATE_KEY` environment variables.
+
+
+### Generating keys
+To create a new private key pair, usee the `generate_keys` command. Three flags must
```

Comment:
Technically two or three right:`--key_name` and (`--print` or [`--out_*`])?

Reply:
reworded